packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -2599,7 +2583,7 @@ class ClipboardStatusNotifier extends ValueNotifier<ClipboardStatus> with Widget
   @override
   void removeListener(VoidCallback listener) {
     super.removeListener(listener);
-    if (!_disposed && !hasListeners) {
```

Comment:
Why aren't these disposed checks needed anymore, was this fixed somewhere in the framework?

Reply:
yes https://github.com/flutter/flutter/pull/108931

Reply:
Thanks :+1: